components/library/libp11/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		libp11
+COMPONENT_VERSION=	0.4.2
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_FMRI=		library/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	System/Security
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	https://github.com/OpenSC/$(COMPONENT_NAME)/wiki
+COMPONENT_ARCHIVE_URL=	https://github.com/OpenSC/$(COMPONENT_NAME)/releases/download/$(COMPONENT_NAME)-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:e2c3614a314b452a9b57e2914252df3ffee59e262dfb75b4fc73a2247f8ddebe
+COMPONENT_LICENSE=		LGPLv2.1+
+COMPONENT_LICENSE_FILE= COPYING
+COMPONENT_DESCRIPTION=	libp11 is a library implementing a thin layer on top of PKCS11 API to make using PKCS11 implementations easier
+COMPONENT_SUMMARY=	libp11 is a library to make using PKCS11 implementations easier
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+COMPONENT_PREP_ACTION = \
+        (cd $(@D) &&\
+        autoreconf -fiv &&\
+        $(RM) config.h )
+
+COMPONENT_PRE_CONFIGURE_ACTION = \
+        ($(CLONEY) $(SOURCE_DIR) $(@D))
+
+INDIR.32=/usr/bin
+BINDIR.64=/usr/bin/$(MACH64)
+LIBDIR.32=/usr/lib
+LIBDIR.64=/usr/lib/$(MACH64)
+
+# Normally this expects /usr/lib/engines - but our openssl does not deliver that
```

Comment:
You keep expecting that the recipe committer knows everything about the packaged bits. I was just unwrapping dependencies wanted by another project to compile ;)

No firm idea about "can" question above, beyond the `README.md` of the project saying it can be used by openssl (transparently with p11-kit, or with extra configuration files if directly).